fury/tests/test_actors.py

```diff
@@ -1294,9 +1294,12 @@ def test_superquadric_actor(interactive=False):
 def test_billboard_actor(interactive=False):
```

Comment:
This test doesn't appear to test anything

Reply:
Thanks for noticing this